app.py

```diff
@@ -1,11 +1,65 @@
-from flask import Flask
+from flask import Flask, render_template, url_for, redirect
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField
+from wtforms.validators import DataRequired
+import random
+import string
 
 app = Flask(__name__)
+app.config.from_pyfile("config.py")
+db = SQLAlchemy(app)
 
-print("Hello, console!")
 
-@app.route('/')
+@app.route("/")
 def index():
-    return "Hello, web!"
+    form = LinkForm()
+    links = Link.query.order_by(Link.link_id.desc()).all()
```

Comment:
You probably don't want to expose this information without authentication?

Reply:
Authentication will be added at a later stage.